core/src/main/java/com/afollestad/materialdialogs/MaterialDialog.kt

```diff
@@ -196,7 +196,8 @@ class MaterialDialog(
         textRes = res,
         text = text,
         fallback = android.R.string.ok,
-        typeface = this.buttonFont
+        typeface = this.buttonFont,
+            textColor = R.attr.md_button_text_color
```

Comment:
Too much indent here

Reply:
Brand new 2.0 user, may be off-

In order to match `md_color_title` and `md_color_content`, would it make sense to call this `md_color_button`? 

Or, possibly:
`md_color_button_positive` and `md_color_button_negative`.